### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by StellaAthena**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,20 +40,19 @@ We offer a choice of layernorm, scalenorm and RMSNorm easily configured by chang
 
 - **Zero Redundancy Optimizer (ZeRO):** 
 
-    - ZeRO stage 1 works seamlessly with NeoX, while ZeRO stage 2 requires pipeline parallelism be set to 0. We are additionally working on integrating ZeRO 3 into the codebase.
+    - ZeRO stage 1 works seamlessly with NeoX, while ZeRO stage 2 requires pipeline parallelism be set to 0, as ZeRO performs pipeline parallelism internally. We are additionally working on integrating ZeRO 3 into the codebase.
```
Comment (sdtblck) [inline README.md:43]: "ZeRO performs pipeline parallelism internally" This is inaccurate

File: README.md [hunk 2]
```diff
@@ -40,20 +40,19 @@ We offer a choice of layernorm, scalenorm and RMSNorm easily configured by chang
...
 
 ## Getting Started
 
-Our codebase relies on [DeeperSpeed](https://github.com/EleutherAI/DeeperSpeed), our fork of the [DeepSpeed](https://github.com/microsoft/DeepSpeed) library with some added changes. 
-We strongly recommend using Anaconda, a virtual machine, or some other form of environment isolation before installing from `requirements.txt`. Failure to do so may cause other repositories that rely on DeepSpeed to break. Python 3.8 or later is required.
+Our codebase relies on [DeeperSpeed](https://github.com/EleutherAI/DeeperSpeed), our fork of the [DeepSpeed](https://github.com/microsoft/DeepSpeed) library with some added changes. We strongly recommend using Anaconda, a virtual machine, or some other form of environment isolation before installing from `requirements/requirements.txt`. Failure to do so may cause other repositories that rely on DeepSpeed to break. Python 3.8 or later is required.
 
-First make sure you are in an environment with `torch>=1.7.1` installed. Then run `pip install -r requirements.txt`. 
+First make sure you are in an environment with `torch>=1.7.1` installed. Then run `pip install -r requirements/requirements.txt`.
```
Comment (sdtblck) [inline README.md:55]: I think the repo needs torch>=1.8 now

File: README.md (hunk 1 above)
Comment (StellaAthena) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -17,13 +17,13 @@ GPT-NeoX is under active development.
...
 
-- GPTNeoX offers full 3D parallelism (data, model and pipeline parallel) using deepspeed, allowing you to scale model training to hundreds of billions of parameters across multiple GPUs.
+- GPTNeoX offers full 3D parallelism (data, model and pipeline parallel) using DeepSpeed, allowing you to scale model training to hundreds of billions of parameters across multiple GPUs.
 
 ### Model Structure
 
 - **Positional Encodings:** 
 
-    - Choose between T5 RPE style positional encodings, a learned encoding added to the input (GPT2-style), Sinusoidal positional encoding, and no positional encodings at all (which [recent](https://arxiv.org/abs/1905.04226) [research](https://arxiv.org/abs/2102.11174) has found to even outperform other positional encodings in autoregressive models).
+    - Choose between T5 RPE style positional encodings, a learned encoding added to the input (GPT2-style), Sinusoidal positional encoding, rotary positional encodings, and no positional encodings at all (which [recent](https://arxiv.org/abs/1905.04226) [research](https://arxiv.org/abs/2102.11174) has found to even outperform other positional encodings in autoregressive models).
```
Comment (kingoflolz) [inline README.md:26]: cite rotary arxiv?

Comment (StellaAthena) [reply README.md]: Added!

**Review by sdtblck — APPROVED** [review]

**Comment by sdtblck** [comment]
I think there's a lot more changes to be made - but we can make those in a separate PR 
this lgtm

**Comment by StellaAthena** [comment]
Yeah, in the week since I posted this things picked up rather quickly.

**Pull request merged**